var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -428,6 +428,7 @@ class Trilinos(CMakePackage, CudaPackage):
     patch('fix_clang_errors_12_18_1.patch', when='@12.18.1%clang')
     patch('cray_secas_12_12_1.patch', when='@12.12.1%cce')
     patch('cray_secas.patch', when='@12.14.1:12.18.1%cce')
+    patch('fix_cxx14_cuda11.patch', when='@13.0.0:13.0.1 cxxstd=14 ^cuda@11:')
```

Comment:
Can you please add a brief comment above the directive containing patch provenance and explaining why the patch is needed? Bonus (not a request) if the patch can be retrieved from remote.

Reply:
I referenced the github issue from trilinos for the NVCC bug with c++14, and a short comment about the CUDA-11 deprecated functions.
I am not sure I understand what you mean with retrieving from remote? I can upload it somewhere, but should I also reference it? My impression was, that the patches are commited to the git repository?

Reply:
> I am not sure I understand what you mean with retrieving from remote? I can upload it somewhere, but should I also reference it?

If patches can be retrieved and applied as is from remote, we prefer that. If they are not we don't require people to upload them somewhere else. Mileage may vary, in the sense that a few k lines patch will very likely be treated differently from a 2 lines patch.